apache/superset
# chore: Add Cypress tests for drill by

### SUMMARY
Tested flows:
- opening the drill by modal -> further drilling -> undo drilling for each supported chart
- open Table view in drill by modal
- verify that Edit chart button URL is correct
- breadcrumbs content
- metadata bar is present

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF


### TESTING INSTRUCTIONS


### ADDITIONAL INFORMATION
